RestaurentManagement/src/main/java/com/restapi/service/MenuService.java

```diff
@@ -42,52 +47,84 @@ public Menu createMenu(Menu menu) {
             }
         }
         menu.setType(existingTypes);
-        return menuRepository.save(menu);
+        return new SuccessDetail(HttpStatus.OK.value(),
+                HttpStatus.OK.getReasonPhrase(),
+                "Create success!",
+                menuRepository.save(menu));
     }
 
-    public List<Menu> getAllMenus() {
-        return menuRepository.findAll();
+    public List<Menu> getAllMenusPaged(int page, int size, String sortBy) {
+        if (page <=0)
+            page = 1;
+        int totalMenus = (int) menuRepository.count();
```

Comment:
You can create a util class(research about it if you don't know it) and create a function to return this result